/**:
  ros__parameters:
    topic_imu: /imu/imu
    acc_ratio: false  
    lidars: ["Ouster"]
    Ouster:
      topic_lidar: /os1_cloud_node1/points
      lidar_type: Ouster
      scan_line: 16
      blind: 4.0
      q_lb: [1.0, 0.0, 0.0, 0.0]      
      t_lb: [0.05, 0.0, -0.055]    
      w_pt: 0.01 
    lidar_time_offset: 0.1

    knot_hz: 100
    if_lidar_only: true  
    
    ds_scan_voxel: 0.5
    point_filter_num: 3
    nn_thresh: 0.5
    coeff_cov: 3.0
    num_nn: 8

    ds_lm_voxel: 0.5
    cube_len: 1000.0

    cov_P0: 0.02
    cov_RCP_pos_old: 0.1
    cov_RCP_ort_old: 0.1
    cov_RCP_pos_new: 1.0
    cov_RCP_ort_new: 1.0
    std_sys_pos: 0.2
    std_sys_ort: 0.1
    cov_acc: [1.0,1.0,1.0]
    cov_gyro: [0.1,0.1,0.1]     
    cov_ba: [0.2, 0.2, 0.2]
    cov_bg: [0.2, 0.2, 0.2]      
    n_iter: 1   
    num_points_upd: 500